src/components/Comments.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from "react";
+import { getCommentsByArticleId } from "../api";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+function Comments() {
+  const { id } = useParams();
+  const [commentsList, setCommentsList] = useState([]);
+
+  useEffect(() => {
+    getCommentsByArticleId(id)
```

Comment:
Need a some way of visually indicating to the user if the data is still being fetched (loading) or if there's an error. Right now it's really not clear if something is broken or not when trying to view your app on slower connections. Check out the notes on useEffect again for an example! 

Generally we should be including this logic anytime a request is made to the API and we don't know how long it will take for that information to come back for the best UX